### ISSUE TITLE
Detailed instruction page

### ISSUE THREAD

**Issue by A1eph0** [llaske/sugarizer#837]
There is minimal instructions currently on how each activity works. A detailed instructions, in the form of an instruction page or an example could help (if not children, then at least their instructors/guardians) to use the program in a better manner, and engage more effectively.

**Comment by llaske** [comment]
Hmmm, did you see that each activity has a tutorial included? Just click on the "?" button in the toolbar and you will have a detailed explanation of activities.
A global overview of activities with a demo video of each is also available [here](https://sugarizer.org/activities.html).

**Comment by hr1sh1-coding** [comment]
Yeah, I am aware of the tutorial included . But the instructions in it is minimal. For some activities it is sufficient, but for others it may not be. For example, in the Abacus tutorial there's no guidance as to how an abacus works or can be used.

I think having more detailed instructions could be significant upgrade to the user experience.

**Comment by llaske** [comment]
We're thinking to write a pedagogic manual for teacher to explain how activities could be useful in the classroom.
I guess it's close to the idea of your improved documentation. 
But to be honest, documentation is more for teacher than for children. Children try and explore and don't read documentation. It's the general philosophy of Sugarizer.
BTW, feel free to suggest improvement on some existing activities tutorial.

**Comment by er-reddyashok** [comment]
can i work on this task

**Comment by llaske** [comment]
@er-reddyashok as I said, I don't think there is room for improvement.
Do you have any idea to improve the current features?

**Comment by er-reddyashok** [comment]
Adding a leaderboard feature to display top performers for daily, weekly, and monthly achievements could be a great way to enhance user engagement on Sugarizer. This feature could motivate users through a sense of accomplishment and friendly competition. Additionally, you might consider customizing the leaderboard to align with educational goals, allowing teachers to highlight achievements based on specific skills or subjects. This could also encourage ongoing use and make learning more interactive and rewarding.

**Comment by llaske** [comment]
Your suggestions are not at all related to this issue.
Plus the concept of leader board means that there is a good way and a wrong way to complete an activity. It's not the case for all Sugarizer activities where we expect some creativity and exploration.
Your suggestion is more related to children evaluation and we've got a Sugarizer Server feature for that with [assignments](https://sugarizer.org/docs/articles/assignments_en.html).

**Comment by SnehaPoojary20** [comment]
I have improved the tutorial descriptions for the Abacus activity. The changes make it easier for beginners and educators to understand.
Here is my PR: https://github.com/sugarlabs/sugarizer/pull/12
Please review it when you get time @llaske .
 Thanks!

**Comment by llaske** [comment]
No interest.
